ui/src/Components/auth/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const loadingImg =
+  "https://cdn.auth0.com/blog/auth0-react-sample/assets/loading.svg";
+
+const Loading = () => (
+  <div className="spinner">
+    {/* <Spinner /> */}
```

Comment:
Is this intentional? Clean up comment if not needed

Reply:
Yes, this is intentional